tools/meson-render-jinja2.py

```diff
@@ -17,7 +17,9 @@ def parse_config_h(filename):
         if not m:
             continue
         a, b = m.groups()
-        if b and b[0] in '0123456789"':
+        # The function ast.literal_eval() cannot be evaluate octal integers, e.g. 0600.
```

Comment:
```suggestion
        # The function ast.literal_eval() cannot evaluate octal integers, e.g. 0600.
```

Reply:
Hmm, we have the following FIXME for dev-kvm-mode:

https://github.com/systemd/systemd/blob/d0a63cf0412ccb9b4edd1f6048e25aa7f68b0133/meson.build#L976

Not sure about the best strategy here.